docs/github/v2/01-getting-started.md

```diff
@@ -109,12 +108,7 @@ have a look at the example below.
 ```yaml
 name: Actions 😎
 
-on:
-  pull_request: {}
-  push: { branches: [main] }
-
-env:
-  UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}
+on: [push, pull_request]
```

Comment:
This will trigger both push inside the pull request and on the main branch (or used to), causing double triggers or unpredictable triggers in PRs.